internal/stacks/stack_manager.go

```diff
@@ -339,13 +359,26 @@ func (s *StackManager) ensureInitDirectories() error {
 	return nil
 }
 
-func (s *StackManager) writeDockerCompose(workingDir string, compose *docker.DockerComposeConfig) error {
-	bytes, err := yaml.Marshal(compose)
+func (s *StackManager) writeDockerCompose(compose *docker.DockerComposeConfig) error {
+	comments := "# This file is generated - DO NOT EDIT!\n# To override config, edit docker-compose.override.yml\n"
+	bytes := []byte(comments)
+	yamlBytes, err := yaml.Marshal(compose)
 	if err != nil {
 		return err
 	}
+	bytes = append(bytes, yamlBytes...)
```

Comment:
I guess this works 🙂

Reply:
Yea... wondered if there was a more efficient way?